### PR TITLE
Feature7

### PR DESCRIPTION
**Pull request by samednayki**
Feature7 interface was designed, click animation was added and navigating to another link was resolved.

### CODE REVIEW

**Review by dogusmiuw — APPROVED** [review]
LGTM!

File: editor-components/feature/feature7/feature7.tsx [hunk 1]
```diff
@@ -1,198 +1,253 @@
...
+    const hasdescription = this.castToString(this.getPropValue("description"));
+    const hasFeatures = this.getPropValue("features");
+    const hasLinks = this.getPropValue("links");
+    const hasTextContent =
+      hasTitle ||
+      hasMainTitle ||
+      hasdescription ||
+      hasLinks ||
+      hasFeatures;
+
```
Comment (Ayazelar) [inline editor-components/feature/feature7/feature7.tsx:167]: rename this variables

File: editor-components/feature/feature7/feature7.tsx [hunk 2]
```diff
@@ -153,17 +153,17 @@ class Feature7 extends BaseFeature {
...
-    const hasMainTitle = this.castToString(this.getPropValue("mainTitle"));
-    const hasdescription = this.castToString(this.getPropValue("description"));
-    const hasFeatures = this.getPropValue("features");
-    const hasLinks = this.getPropValue("links");
+    const title = this.castToString(this.getPropValue("title"));
+    const mainTitle = this.castToString(this.getPropValue("mainTitle"));
+    const description = this.castToString(this.getPropValue("description"));
+    const features = this.getPropValue("features");
+    const links = this.getPropValue("links");
     const hasTextContent =
```
Comment (Ayazelar) [inline editor-components/feature/feature7/feature7.tsx:161]: check line 179. you are using "hasTextContent.length". It has a bit of a strange spelling

**Review by Ayazelar — CHANGES_REQUESTED** [review]
change

**Review by dogusmiuw — APPROVED** [review]

**Review by Ayazelar — APPROVED** [review]

**Pull request merged**